src/Entity/Slot.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity;
+
+use Acquia\LiftClient\Entity\Visibility;
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\acquia_lift\SlotInterface;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines the Drupal created slots.
+ *
+ * @ConfigEntityType(
+ *   id = "acquia_lift_slot",
```

Comment:
Drupal auto-detects this namespace so changing acquia_lift.slot.yml would cause for Drupal to no longer find this file. I think it takes the Classname and the module name. This makes sense as it then will be unique to that module, and the id will be unique towards the complete drupal scope. This should not be changed. Renaming is great but this will break the module. The only alternative we can do is to rename the class from Slot.php to AcquiaLiftSlot.php and then the config file becomes acquia_lift.acquia_lift_slot I think. Happy to accept pull requests if you manage to clean this up further.